documentation/home.md

```diff
@@ -0,0 +1,30 @@
+# MuleSoft AI Chain Connector Overview
+
+The MuleSoft AI Chain Connector is designed to help developers easily build and manage AI-driven agents within MuleSoft Anypoint Platform. It provides the tools and support needed to integrate Large Language Models (LLMs), embeddings, and other advanced AI services into MuleSoft applications.
+
+## Key Features
+
+The MuleSoft AI Chain Connector simplifies AI integration into MuleSoft applications with:
+
+- Seamless Interaction with LLMs: Effortlessly integrate and utilize large language models (LLMs) for tasks such as natural language processing, text generation, and more, including advanced features like text generation, analysis, and complex language tasks.
+- Embeddings and Search: Leverage embeddings to handle tasks like text similarity, document search, and clustering, all within MuleSoft applications.
+- Optimized Performance: Designed for high efficiency and performance in enterprise-grade MuleSoft applications, ensuring smooth handling of AI operations.
+- File-Based Local Vector Stores: Create simple, file-based vector stores directly within the connector, ideal for local use or POC designs. For external vector store integration, use the dedicated Vector Store Connector.
+- Comprehensive AI Tools and Services: Access a wide array of AI-driven features, including Retrieval-Augmented Generation (RAG) for document retrieval, dynamic tool integration (Function Calling), and image model support for tasks like recognition and manipulation.
+
+## Integration with the MuleSoft Ecosystem
+
+The MuleSoft AI Chain Connector integrates tightly with the MuleSoft ecosystem, offering:
+
+- Customizable Tooling: Use configuration files for flexible setups tailored to your needs. 
+- Easy Tooling Management: Access and manage a wide range of AI tools directly through Anypoint Exchange.
+
+## Enhanced Capabilities using Anypoint Platform
+
+By leveraging MuleSoft’s powerful Anypoint Platform, the MAC Connector enhances AI projects with:
+
+- AI Lifecycle Management: Manage the entire lifecycle of AI agents, from design to deployment, using Anypoint Platform.
+- Centralized AI Design: Design agents using Anypoint Design Center for seamless workflow integration.
```

Comment:
Not sure if we want to explicitly call out Design Center since it's not getting a ton (or any) investment